Point Quickstart option 2 to Barge repo for now

ttmc opened the pull request:
Quickstart option 2 isn't fully working right now because Barge isn't working fully right now, but as soon as it is, the Barge `README.md` file will be the first to be updated. Later, we can update the Quickstart page in the docs (which has a boiled-down version of the Barge `README.md` file).

For now, option 2 mainly just points to the Barge repo (i.e. the Barge `README.md` file).

Related: https://github.com/oceanprotocol/barge/pull/119

kremalicious commented:
With this PR we send first-time users on a treasure hunt for the best option and they need to decide what that is. Quickstart already has 2 options, the readme then has many more.

Why not simply leave the instructions on here, and assume we fix the problems? Cause instructions are how it is supposed to work. Removing the instructions and pointing people to the repo and long readme when something is broken defeats the purpose of the docs I think. All the other copy changes are cool though.

Maybe we simply add a note at bottom about the alpha state?

ttmc commented:
@kremalicious I brought back option 2 much as it was before but:
- added an initial warning that it might not work (yet)
- removed the part about having to set up Azure Storage first because that's not true any more
